www/css/style.css

```diff
@@ -836,7 +852,7 @@ a.list-text {
 }
 
 button.button.ng-binding {
-  font-size: 15px;
+  font-size: 12px;
```

Comment:
again, comment on which buttons this affects...

Reply:
This affected the Label buttons.

Reply:
@sebastianbarry again, create a new class since this looks like it will change the size for all buttons.

Reply:
I don't see any difference between the `15px` and the `12px` when I change back and forth between them (in the diary or otherwise). I also don't see anywhere this is being used in the diary screen, so this change seems non-consequential.

I am switching it back to 15px

_**Confirming this change as complete**_